kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDetailsController.java

```diff
@@ -208,36 +212,52 @@ private void initialize() {
 
         // capture descriptions information
         StringProperty fqnTextProperty = patternViewModel.getProperty(FQN_DESCRIPTION_NAME_TEXT);
-        fqnText.textProperty().bind(fqnTextProperty);
+        latestFqnText.textProperty().bind(fqnTextProperty);
         addFqnMenuItem.setVisible(fqnTextProperty.isEmpty().get());
```

Comment:
Can the menu visible property be bound based on the following property?
```java
ObjectProperty<DescrName> fqnNameProp = patternViewModel.getProperty(FQN_DESCRIPTION_NAME);
addFqnMenuItem.visibleProperty().bind(fqnNameProp.isEmpty());
```
This is off the top of my head. Please verify code above.

Reply:
Yes have already done that in the latest update...

Reply:
My suggestion is more concise. The code above is redundant and calling `getNameText()` when it's a raw value. 
```java
addFqnMenuItem.visibleProperty().bind(fqnNameProp.isEmpty());
```